legal-api/src/legal_api/models/filing.py

```diff
@@ -112,6 +112,13 @@ class Source(Enum):
                 'BEN': 'BCINC'
             }
         },
+        'conversion': {
+            'name': 'conversion',
+            'title': 'Conversion Ledger',
+            'codes': {
+                'BEN': 'BCINC'
```

Comment:
Shouldn't this be for the existing types? There should be no conversion ledgers for BEN.

Reply:
Looking into this a little closer and I realize this `FILINGS` dictionary is used as a mechanism to retrieve fee codes which is eventually passed to the payment api.  I'll remove this.  I must have been a little careless with copy/pasting from IA code above.

Reply:
removed conversion ledger entry from `FILINGS` dictionary as per previous comment.